recipe/meta.yaml

```diff
@@ -6,23 +6,36 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/omegaconf-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: d5d4b6d29955cc50ad50c46dc269bcd92c6e00f5f90d23ab5fee7bfca4ba4cc7
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  # skip s390x because openjdk is not available
+  skip: true  # [s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
-    - antlr 4.9
```

Comment:
conda-forge was able to build it with antlr 4.9 https://github.com/conda-forge/omegaconf-feedstock/blob/c7922702617e04c8cda0d013131c6b82d922134c/recipe/meta.yaml#L21. Are we able to do the same?

Reply:
We do not have it, we should produce it first. https://anaconda.org/anaconda/antlr

I would be glad to add `antlr` to our schedule.

In case we add it, then we do not need to add `openjdk`, but we would probably need to patch the source here:
https://github.com/omry/omegaconf/blob/350bdb632865c5dd2286f2f6521acefe4abd843d/build_helpers/build_helpers.py#L30-L40

Which seems cumbersome.

In the next version, they remove the `.jar` and the `java` invocation and include `antlr` in a vendor folder, we could have a look at how they change this and apply a patch. But still, it's a consistent change to the source.

Did I get it wrong?

Reply:
You got it right. There is no point in un-vendoring antlr as it is used as a build tool only (i.e. translating grammars to python code at build time).